src/components/Posts/ComposeReply.tsx

```diff
@@ -112,9 +132,30 @@ const ComposeReply = ({
               <IconButton size="small">
                 <AddPhotoAlternateOutlinedIcon />
               </IconButton>
-              <IconButton size="small">
+              <IconButton
+                size="small"
+                onClick={toggleEmojiPicker}
+                id="emoji-button"
+              >
                 <EmojiEmotionsOutlinedIcon />
               </IconButton>
+              {showEmojiPicker && (
+                <Box
+                  sx={styles.emojiContainer}
+                  ref={emojiContainerRef}
+                  tabIndex={0}
+                  onBlur={clickOffEmojiPicker}
+                >
+                  <EmojiPicker
```

Comment:
This library also has a prop called `emojiStyle` where it is set default to value `apple`. If we want it to look more like twitter, we should set the value to `twitter`. The way I suggested it to be done does not have to be the only way. If you want to create a const for EmojiStyle.TWITTER and then use that, it'll look better as well. Up to you
```suggestion
                  <EmojiPicker
                    emojiStyle={EmojiStyle.TWITTER}
```

Reply:
I used the twitter emoji style.